### DISCUSSION TITLE
Hope suggestion about how to generate the mesh for SU2

### DISCUSSION THREAD

**Discussion by zhang-qiang-github** [su2code/SU2#1890] [Q&A]
I want to implement `CFD` simulation with `SU2`. 

A tube is used to simulate vessel, and it looks like:

![image](https://user-images.githubusercontent.com/29271051/212541021-51f12b0c-a22d-4485-84fe-507416d5cee8.png)

I hope the wall should be red color part, and inlet/outlet should be yellow color part:

![image](https://user-images.githubusercontent.com/29271051/212543758-b38e0276-172c-4909-aa90-1b1cbfaa29b4.png)

Then,  tetrahedron is generated from this `mesh`. 

![image](https://user-images.githubusercontent.com/29271051/212543281-2547537b-8473-4af8-b14a-80be177f1820.png)

My question is: how to build `inlet/outlet/wall` from the geneated tetrahedron? Suggestions is needed to generate the `SU2` mesh. Any suggestion is appreciated~~~

**Comment by bigfooted** [comment]
What is your meshing tool? Usually, the inlet, outlet and wall are independent surface entities and you can select them, give them a name, and mark them as a boundary.

**Reply by zhang-qiang-github** [reply]
I will try `gmsh` to generate my `su2` file. Many thanks for kindly reply~~~
